content/shmem_alltoall.tex

```diff
@@ -103,7 +103,7 @@
     This routine restores \VAR{pSync} to its original contents.  Multiple calls
     to \openshmem\ routines that use the same \VAR{pSync} array do not require
     that \VAR{pSync} be reinitialized after the first call.
-    You must ensure that the \VAR{pSync} array is not being updated by any
+    Ensure that the \VAR{pSync} array is not being updated by any
```

Comment:
How about "The user must ensure..."?  We use similar phrasing elsewhere in the document.